src/Microsoft.Extensions.Logging.Testing/AssemblyTestLog.cs

```diff
@@ -20,6 +22,15 @@ public class AssemblyTestLog : IDisposable
         public static readonly string OutputDirectoryEnvironmentVariableName = "ASPNETCORE_TEST_LOG_DIR";
         private static readonly string LogFileExtension = ".log";
         private static readonly int MaxPathLength = 245;
+        private static char[] InvalidFileChars =
+            Path.GetInvalidPathChars()
+            .Union(Path.GetInvalidFileNameChars())
+            .Union(new char[] {
+                ' ', // space
+                '\\', // back slash
+                '/', // forward slash
```

Comment:
Isn't this already an InvalidFileNameChar on every platform? Maybe a whitelist would be better.

Reply:
I think so. I prefer not using a whitelist since that probably takes more work.